src/app/shared/cookies/orejime-configuration.ts

```diff
@@ -0,0 +1,182 @@
+import {
+  IMPERSONATING_COOKIE,
+  REDIRECT_COOKIE,
+} from '../../core/auth/auth.service';
+import { TOKENITEM } from '../../core/auth/models/auth-token-info.model';
+import {
+  CAPTCHA_COOKIE,
+  CAPTCHA_NAME,
+} from '../../core/google-recaptcha/google-recaptcha.service';
+import { LANG_COOKIE } from '../../core/locale/locale.service';
+import { NativeWindowRef } from '../../core/services/window.service';
+
+/**
+ * Cookie for has_agreed_end_user
+ */
+export const HAS_AGREED_END_USER = 'dsHasAgreedEndUser';
+
+export const ANONYMOUS_STORAGE_NAME_OREJIME = 'orejime-anonymous';
+
+export const GOOGLE_ANALYTICS_OREJIME_KEY = 'google-analytics';
+
+/**
+ * Orejime configuration
+ * For more information see https://github.com/empreinte-digitale/orejime
+ */
+
+export function getOrejimeConfiguration(_window: NativeWindowRef): any {
+  return {
+    storageName: ANONYMOUS_STORAGE_NAME_OREJIME,
```

Comment:
Based on the comments above, I don't believe this setting is working.  It's also not listed in the Orejime config reference, so it may not be valid. https://github.com/empreinte-digitale/orejime?tab=readme-ov-file#configuration

Maybe it should be removed?  

UPDATE: It looks like Orejime renamed this to be `cookieName`.  That might be why it's not working.

Reply:
@tdonohue, good catch! With that I can see both orejime-anonymous and orejime-<uuid>!

![image](https://github.com/user-attachments/assets/af2b312d-9b78-41d0-9392-c2b51e67fb2a)

But tests are still failing, I'm on them.